### PR TITLE
Update team in FAQ

### PR DESCRIPTION
**Pull request by merelcht**
Signed-off-by: Merel Theisen <merel.theisen@quantumblack.com>

## Description
- I finally managed to merge my Github account with my personal one, so can use my personal username now 🙂 
- I then noticed several team members missing in the FAQ page so updated that

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1973"></a>

### CODE REVIEW

File: docs/source/faq/faq.md [hunk 1]
```diff
@@ -16,15 +16,18 @@ Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) a
 Currently, the core Kedro team consists of
 [Ahdra Merali](https://github.com/AhdraMeraliQB),
 Andrew Mackay,
```
Comment (jmholzer) [inline docs/source/faq/faq.md:18]: I think Mackay's GH is https://github.com/Mackay031, unless there's a reason he doesn't want this here?

**Review by jmholzer — APPROVED** [review]
Thanks, LGTM!

File: docs/source/faq/faq.md (hunk 1 above)
Comment (merelcht) [inline docs/source/faq/faq.md:18]:
```suggestion
[Andrew Mackay](https://github.com/Mackay031),
```

Comment (merelcht) [reply docs/source/faq/faq.md]: Good shout, I've added it!

**Review by stichbury — APPROVED** [review]
🏆 🌟

**Pull request merged**